zstacklib/zstacklib/utils/lvm.py

```
@@ -2774,6 +2775,142 @@ def get_lv_location(lv_path):
+                        (command, cmd.stdout))
+
+
+def _get_vg_range_report(vg_uuid):
+    return _run_lvm_json_report(
+        "vgs --readonly --nolocking -t --units b --nosuffix --reportformat json "
+        "-o vg_name,vg_uuid,vg_extent_size,vg_seqno %s" % linux.shellquote(vg_uuid))
+
+
+def _get_vg_seqno(vg_report):
+    rows = []
+    for report in vg_report.get("report", []):
```

Ruff (0.16.0)

[error] Import `report` from line 17 shadowed by loop variable

(F402)